packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -183,10 +183,6 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
       setTimeout(() => this._previousActiveElement!.focus(), 0);
     }
 
-    if (this.props.onMenuDismissed) {
-      this.props.onMenuDismissed(this.props);
-    }
-
```

Comment:
Same here, why are we removing this?

Reply:
This is mostly to avoid confusion between the prop in ContextualMenu and BaseButton

Reply:
Won't having onMenuDismissed as a prop on the BaseButton and a prop on ContextualMenu confuse things just as much, if not more?